tests/src/Phansible/Roles/RoleTest.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Phansible\Roles;
+
+class RoleTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @use Phansible\Roles\BaseRole::getInitialValues
+     */
+    public function testInitialValuesShouldReturnArray()
+    {
+        $role = $this->getMockBuilder('\Phansible\BaseRole')
+            ->disableOriginalConstructor()
+            ->getMockForAbstractClass();
+
+        $role->expects($this->once())
+            ->method('getInitialValues')
+            ->will($this->returnValue([]));
+
+        $this->assertInternalType('array', $role->getInitialValues());
```

Comment:
This test makes no sense.

You told the baserole to return empty array and then you assert that returns an array.
What you win checking this?